gulpfile.js

```diff
@@ -69,23 +65,18 @@ gulp.task('lint', function() {
     .pipe(eslint.failOnError());
 });
 
-gulp.task('test', ['clean'], function() {
-  return gulp.src(KARMA_FILES)
-    .pipe(karma(karmaConfig('run')))
-    .on('error', function(err) {
-      throw err;
-    });
-});
-
 gulp.task('clean', function() {
   return gulp.src('test/coverage')
     .pipe(clean());
 });
 
-gulp.task('coveralls', ['test'], function() {
-  return gulp.src(['test/coverage/**/lcov.info'])
-    .pipe(coveralls());
-});
+gulp.task('test', gulp.series('clean', function() {
```

Comment:
Ugrade to gulp 4.0 deprecated `gulp.task('coveralls', ['test'], function() { ... ` (which reads, for the task 'coveralls', do the 'test' task, then perform this function) in favor of `gulp.series('coveralls', function ...`